### PR TITLE
formulae(a-g): standardize autoreconf and configure steps [p1]

### PR DESCRIPTION
**Pull request by chenrui333**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/a/advancecomp.rb [hunk 1]
```diff
@@ -24,9 +24,8 @@ class Advancecomp < Formula
   uses_from_macos "zlib"
 
   def install
-    system "autoreconf", "-fiv"
-    system "./configure", "--disable-debug", "--disable-dependency-tracking",
-                          "--enable-bzip2", "--prefix=#{prefix}", "--mandir=#{man}"
+    system "autoreconf", "--force", "--install", "--verbose"
```
Comment (SMillerDev) [inline Formula/a/advancecomp.rb:27]: Do we have an audit for this?

File: Formula/b/bash-completion@2.rb [hunk 2]
```diff
@@ -40,7 +40,7 @@ def install
       s.gsub! "(/etc/bash_completion.d)", "(#{etc}/bash_completion.d)"
     end
 
-    system "autoreconf", "-i" if build.head?
+    system "autoreconf", "--force", "--install", "--verbose" if build.head?
     system "./configure", "--prefix=#{prefix}"
```
Comment (SMillerDev) [inline Formula/b/bash-completion@2.rb:44]: Why not this one?

File: Formula/c/cdparanoia.rb [hunk 3]
```diff
@@ -61,7 +61,7 @@ def install
                 "-Wl,-rpath,#{Formula["cdparanoia"].libexec} -L ../interface"
     end
 
-    system "autoreconf", "-fiv"
+    system "autoreconf", "--force", "--install", "--verbose"
     system "./configure", "--prefix=#{prefix}",
```
Comment (SMillerDev) [inline Formula/c/cdparanoia.rb:65]: Why not this one?

**Review by SMillerDev — COMMENTED** [review]
I think it might be good to add comment when the `std_configure_args` can't be used

File: Formula/a/advancecomp.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/a/advancecomp.rb:27]: we dont have audit for this, but I can add one later.

File: Formula/b/bash-completion@2.rb (hunk 2 above)
Comment (chenrui333) [reply Formula/b/bash-completion@2.rb:44]: yeah, technically we can, I will update.

File: Formula/a/advancecomp.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/a/advancecomp.rb:27]: yeah, totally.

Comment (chenrui333) [reply Formula/a/advancecomp.rb:27]: will follow up on this.

**Review by daeho-ro — APPROVED** [review]

**Pull request merged**